enhancements/sig-policy/28-selective-policy-enforcment/README.md

```diff
@@ -0,0 +1,639 @@
+    child policies created
++ A policy bound to the same cluster under multiple PlacementBindings
+  will be set to `enforce` if _any_ of those bindings has
+  `remediationActionOverride.remediationAction` set to enforce.
++ A policy with `remediationAction` already set to `enforce` is
+  unaffected by a PlacementBinding with
+  `remediationActionOverride`.
+
+### Sub filtering option in PlacementBinding
+The `subFilter` field allows users to control what clusters can be
+considered when evaluating this PlacementBinding. This field is a
+boolean with valid values of `subFilter: [false|true]`. To preserve
+the existing semantics of multiple placement bindings attached to a
+single policy this option defaults to `false`.
+
+The behavior of a PlacementBinding with `subFilter: true` is:
+
++ For the bound Policy or PolicySet, only clusters selected by another
+  PlacementBinding where `subFilter` is false will be considered for
+  binding by this PlacementBinding. In other words, a PlacementBinding
+  with `subFilter: true` will not bind any additional clusters to the
+  policy.
+
+The behavior of a PlacementBinding with `subFilter: false` is
+unchanged from the current PlacementBinding behavior.
+
+Including the `subFilter` at the top level of the CR allows for
+additional override actions or transformations of the bound policy in
+the future to make use of the same subFiltering functionality.
+
+When `subFilter` is combined with the `remediationActionOverride` the
+behavior is:
+
++ A PlacementBinding without
+  `remediationActionOverride.remediationAction` and with
+  `subFilter:false` (which are the defaults for both) behaves exactly
+  as it does without this enhancement.
++ A PlacementBinding without
+  `remediationActionOverride.remediationAction` and with
+  `subFilter:true` is effectively a "no-op". No additional clusters
+  are bound. No override is performed.
++ A PlacementBinding with
+  `remediationActionOverride.remediationAction: enforce` and with
+  `subFilter:false` will override the remediationAction of all
+  policies bound by this PlacementBinding.
++ A PlacementBinding with
+  `remediationActionOverride.remediationAction: enforce` and with
+  `subFilter:true` will override the remediationAction of any policies
+  bound by both this PlacementBinding _and_ at least one `subFilter:
+  false` PlacementBinding.
+
+The subFilter option is specified to allow a user to ensure that a
+single PlacementBinding can be used to define the set of potentially
+affected clusters. Without the subFilter option, it is possible to
+erroneously select additional clusters in the "override"
+PlacementBinding and accidentally enforce the policy to those
+additional clusters. The `subFilter` option, when set true, ensures
+that a second PlacementBinding used to enforce the policy to a smaller
+number of clusters will respect the intent of the initial binding.
+
+For a given Policy, when there are multiple PlacementBindings
```

Comment:
I wonder if we should add a diagram for this, I would be happy to draft something (I did it on paper and it certainly helped me understand what this is saying)

Reply:
I think that would be really helpful. Thanks!

Reply:
Thoughts? Easy to change:

![override-diagram](https://user-images.githubusercontent.com/44813129/183670708-2be8fd33-375d-41b7-8bf5-19b7bb9b999a.png)

Reply:
Looks great. Thank you! If you want to post a PR to this branch I'll merge it, or I can directly add the diagram.

Reply:
You can add it - no need for an extra commit

Reply:
Added. Thank you!